src/main/scala/org/broadinstitute/dsde/firecloud/service/FireCloudDirectives.scala

```diff
@@ -27,23 +26,22 @@ trait FireCloudDirectives extends spray.routing.Directives with PerRequestCreato
   def passthrough(unencodedPath: String, methods: HttpMethod*): Route =
     passthrough(Uri(unencodedPath), methods:_*)
 
-  def passthrough(uri: Uri, methods: HttpMethod*): Route =
-    passthrough(requestCompression = true, uri, methods:_*)
-
   def passthrough(requestCompression: Boolean, unencodedPath: String, methods: HttpMethod*): Route =
```

Comment:
this is the one you (rightfully) suggest should go away, correct? Fine to leave it in for now - either clean it up as another PR on this story, or file a new tech debt story to clean up later (I can help with filing a ticket if you need it)

Reply:
I already did the work to remove it as it was straightforward.

Just so I know, do we have a criterion for situations like this, such as 'do it if it takes less than an hour'? Should they be accompanied by a ticket even if they take less than an hour?

Reply:
There's no concrete guideline for this. It's really developer prerogative, and you'll get a good feel for this over time. In the meantime, reach out to peers and and get opinions! As far as a separate ticket ... to me, that's based on the risk of regression and pervasiveness of the change. If it's risky and needs deeper QA/review, make a ticket. If it's simple, just include it in some other PR.